gfxr_ext/decode/dive_file_processor.cpp

```diff
@@ -119,6 +121,7 @@ bool DiveFileProcessor::ProcessStateMarker(const format::BlockHeader& block_head
         state_end_marker_file_offset_ = TellFile(gfxr_file_name_);
         GFXRECON_LOG_INFO("Stored state end marker offset %d", state_end_marker_file_offset_);
         GFXRECON_LOG_INFO("Single frame number %d", GetFirstFrame());
+        DivePM4Capture::GetInstance().MaybeStartCapture();
```

Comment:
should we avoid calling this (mainly for avoid calling DivePM4Capture::DivePM4Capture() ) when pm4 capture is not enabled for the Replay session?